modules/manage/partials/tiered-storage.adoc

```diff
@@ -881,7 +881,12 @@ rpk topic alter-config <topic_name> --set redpanda.remote.read=true --set redpan
 
 Topic-level properties override cluster-level properties. For example, for new topics, if `cloud_storage_enable_remote_write` is set to `true`, you can set `redpanda.remote.write` to `false` to turn it off for a particular topic.
 
-NOTE: If you disable remote writes and reads on a topic, that is, set `redpanda.remote.read` and `redpanda.remote.write` to `false`, the data that has been offloaded to object storage is not deleted. When you re-enable remote writes and reads, Redpanda resumes uploading data to object storage starting from the last write point, and the part of the log uploaded to object storage becomes available for reads again.
+[NOTE]
+====
+If you disable remote writes and reads on a topic, that is, set `redpanda.remote.read` and `redpanda.remote.write` to `false`, the data that has been offloaded to object storage is not deleted. When you re-enable remote writes and reads, Redpanda resumes uploading data to object storage starting from the last write point, and the part of the log uploaded to object storage becomes available for reads again.
+
+Redpanda strongly recommends that you do not re-enable Tiered Storage after previously enabling and disabling it, because this can result in inconsistent data and data gaps in Tiered Storage for a topic.
```

Comment:
> If you disable remote writes and reads on a topic, that is, set `redpanda.remote.read` and `redpanda.remote.write` to `false`, the data that has been offloaded to object storage is not deleted. When you re-enable remote writes and reads, Redpanda resumes uploading data to object storage starting from the last write point, and the part of the log uploaded to object storage becomes available for reads again.

This implies TS is enabled, disabled, and can be enabled once again. This is conflicting with the new warning we want to add. Probably the current NOTE should be completely deleted. @mattschumpert thoughts?

Reply:
Exactly -- my preference would be to remove the note entirely.

Reply:
@daisukebe Matt confirmed via Slack that we should remove this note.